backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -89,6 +89,41 @@ public async Task<IQueryable<Organization>> AddProjectToOrg(
         return dbContext.Orgs.Where(o => o.Id == orgId);
     }
 
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> AddProjectsToOrg(
+        LexBoxDbContext dbContext,
+        IPermissionService permissionService,
+        [Service] ProjectService projectService,
+        Guid orgId,
+        IEnumerable<Guid> projectIds)
```

Comment:
I'm getting a warning about multiple enumerations for this variable, it would be better to just make it an array, that's what we've done on other apis

Reply:
Done.